node/utils/src/lib.rs

```diff
@@ -66,3 +66,29 @@ where
     let new_struct: S = toml::from_str(toml_string)?;
     Ok(new_struct)
 }
+
+/// Get file count in a certain directory
+/// Will return the number of files in the directory
+///
+/// # Panics
+/// It will panic if:
+/// - The provided path doesn't exist.
+/// - The process lacks permissions to view the contents.
+/// - The path points at a non-directory file.
```

Comment:
Remove this when panic removed in function

Reply:
ok